### PR TITLE
Updating the doc

### PR DESCRIPTION
**Pull request by rybus**

### CODE REVIEW

File: Resources/doc/Getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
++---------------------------------------+-------------------------------------+
+| File                                  | Description                         |
++=======================================+=====================================+
+| **attribute\_groups\_accesses.yml**   | Contains ACL for attribute groups   |
++---------------------------------------+-------------------------------------+
+| **category\_accesses.yml**            | Contains ACL for categories         |
++---------------------------------------+-------------------------------------+
+| **locale\_accesses.yml**              | Contains ACL for locales            |
++---------------------------------------+-------------------------------------+
+
```
Comment (fitn) [inline Resources/doc/Getting-started.rst]: Missing jobs_accesses.yml

File: Resources/doc/Getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
++------------------------+-------------------------------------------------------------------------+
+| **user\_groups.yml**   | Define all user groups (code + label)                                   |
++------------------------+-------------------------------------------------------------------------+
+| **user\_roles.yml**    | Define all user roles (code + label)                                    |
++------------------------+-------------------------------------------------------------------------+
+| **users.yml**          | Define users list                                                       |
++------------------------+-------------------------------------------------------------------------+
+
+You can still have a look on the `Akeneo PIM minimal
+fixtures <https://github.com/akeneo/pim-community-dev/tree/master/src/Pim/Bundle/InstallerBundle/Resources/fixtures/minimal>`__
```
Comment (fitn) [inline Resources/doc/Getting-started.rst]: I think it should come from 1.3 branch

File: Resources/doc/Getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+details on how to customize the init.xslx file.
+
+Change PIM parameter to use your custom installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: yml
+
+    parameters:
+        pim_installer.fixture_loader.job_loader.config_file: 'PimExcelConnectorBundle/Resources/config/fixtures_jobs.yml'
+
```
Comment (fitn) [inline Resources/doc/Getting-started.rst]: How can I do that? I think it should be explained ;)

File: Resources/doc/Getting-started.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+Define the data used by the installer :
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: yml
+
+    # app/config/pim_parameters.yml
+    parameters:
+        ...
+        installer_data: 'AcmeDemoBundle:minimal'
```
Comment (fitn) [inline Resources/doc/Getting-started.rst]: This should not be changed in pim_parameters.yml but override in parameters.yml.dist to be added in parameters.yml

File: Resources/doc/Metric-types.rst [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- ``Area``
+- ``Binary``
+- ``Frequency``
+- ``Length``
+- ``Power``
+- ``Speed`` 
+- ``Temperature``
+- ``Volume`` 
+- ``Weight``
```
Comment (fitn) [inline Resources/doc/Metric-types.rst]: Why does it miss some MeasureBundle families?

File: Resources/doc/Categories-tab.rst [hunk 6]
```diff
@@ -0,0 +1,23 @@
+Categories tab
+==============
+
+Category code
+-------------
+
+Attribute code must be only composed of letters, numbers, spaces and
```
Comment (fitn) [inline Resources/doc/Categories-tab.rst]: Category code?

File: Resources/doc/Channels-tab.rst [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+============
+
+In this tab you can define all the channels you are using. For more
+details about the key concept of channels, see `channels in the user
+guide <http://www.akeneo.com/doc/user-guide/key-concepts/channels/>`__.
+
+Channel code
+------------
+
+Attribute code must be only composed of letters, numbers, spaces and
```
Comment (fitn) [inline Resources/doc/Channels-tab.rst]: Channel code?

File: Resources/doc/Channels-tab.rst [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+Channel locales
+---------------
+
+List here the comma-separated locales that compose your channel.
+
+Channel currencies
+------------------
+
+List here the comma-separated locales that compose your currencies.
```
Comment (fitn) [inline Resources/doc/Channels-tab.rst]: I don't get it..!

File: Resources/doc/Getting-started.rst (hunk 4 above)
Comment (rybus) [inline Resources/doc/Getting-started.rst]: @fitn I didn't understand :/ the parameters.yml.dist already contains that statement so it is just about changing it right ?

File: Resources/doc/Attribute-groups.rst [hunk 9]
```diff
@@ -0,0 +1,39 @@
+Attribute groups tab
+====================
+
+An attribute group is composed of attributes. Attributes groups are
+shown on the left menu of the product edit form.
+
+Group code
```
Comment (fitn) [inline Resources/doc/Attribute-groups.rst]: Attribute Group code

File: Resources/doc/Metric-types.rst (hunk 5 above)
Comment (rybus) [inline Resources/doc/Metric-types.rst]: if there are missing in this list, they are missing in the init.xslx file too. I should add'em inside the file as well

**Comment by fitn** [comment]
Don't hesitate to add Troubleshooting and bug and issues section ;)

File: Resources/doc/Attribute-groups.rst (hunk 9 above)
Comment (rybus) [inline Resources/doc/Attribute-groups.rst]: nope, it is not a mistake this time :p

File: Resources/doc/Getting-started.rst (hunk 3 above)
Comment (rybus) [inline Resources/doc/Getting-started.rst]: ok, added explaination just before !

**Pull request closed without merging**